### PR TITLE
Add small top skinny banners on code.org

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds a new `top_skinny_banner` view for use across code.org, and specifically added to [code.org/csd](http://code.org/csd) and [code.org/educate/curriculum/middle-school](https://code.org/educate/curriculum/middle-school) in this PR.
- Allows for a light or dark option based on a page's UI needs 
- When the close button is clicked, the banner is hidden and uses `localStorage` to keep it hidden ([ref](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage))
- Fully responsive and keyboard/screen reader navigable 
- Works with LTR and RTL languages
- Also fixed the path on some site-wide JS that adds a width to all icon (`<i>`) elements ([see file](https://github.com/code-dot-org/code-dot-org/pull/56910/files#diff-7a789de52a116115d4b12143392501e53b981c64eab13e29f80731084d90a76f))
  - This was breaking on pages within subdirectories so the help icon in the top nav (`?`) and the close icon in this skinny banner (`x`) were unaligned on [code.org/educate/curriculum/middle-school](https://code.org/educate/curriculum/middle-school)

**Jira ticket:** [ACQ-1507](https://codedotorg.atlassian.net/browse/ACQ-1507)

----

## Light version
Light

## Dark version
Dark

## Hiding the banners

https://github.com/code-dot-org/code-dot-org/assets/9256643/47ea0a1b-c206-46ec-a321-d3c1b2490cb3

## Responsive

https://github.com/code-dot-org/code-dot-org/assets/9256643/9d4e6fa9-f008-4e9e-8f86-76400a1c9f9d

### CODE REVIEW

File: pegasus/sites.v3/code.org/styles_min/040-page.css [hunk 1]
```diff
@@ -8,31 +8,31 @@
 
 :root {
   /* Colors */
-  --brand_primary_light: #ABDFE5;
-  --brand_primary_medium: #7BC6CF;
-  --brand_primary_default: #0093A4;
+  --brand_primary_light: #abdfe5;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/styles_min/040-page.css:11]: This file got linted, sorry for the extra lines! 😞

File: pegasus/sites.v3/code.org/styles_min/040-page.css [hunk 2]
```diff
@@ -573,6 +573,75 @@ nav.main #hamburger #hamburger-icon {
   }
 }
 
+/* Start top skinny banner styles --------------- */
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/styles_min/040-page.css:576]: These are the relevant style changes to this PR.

**Comment by dju90** [comment]
Hi Kelby, this looks great! Do you know how easy it would be to duplicate or extend this component for studio.code.org (e.g. logged-in pages, the dashboard, etc.)? We could really use this in a lot of scenarios when a user is logged in.

**Comment by kelbyhawn** [comment]
> Hi Kelby, this looks great! Do you know how easy it would be to duplicate or extend this component for studio.code.org (e.g. logged-in pages, the dashboard, etc.)? We could really use this in a lot of scenarios when a user is logged in.

Thank you! I think it should be pretty easy to make a component for studio — @dmcavoy @bethanyaconnor could we add this as one of my internship tasks?

**Review by TurnerRiley — APPROVED** [review]
LGTM!

**Comment by dmcavoy** [comment]
> > Hi Kelby, this looks great! Do you know how easy it would be to duplicate or extend this component for studio.code.org (e.g. logged-in pages, the dashboard, etc.)? We could really use this in a lot of scenarios when a user is logged in.
> 
> Thank you! I think it should be pretty easy to make a component for studio — @dmcavoy @bethanyaconnor could we add this as one of my internship tasks?

I wonder if this should be something that Denys does as part of the design system work. @bethanyaconnor what do you think?

**Pull request merged**